app/javascript/crayons/Modal/Modal.jsx

```diff
@@ -17,6 +17,21 @@ function getAdditionalClassNames({ size, className }) {
   return additionalClassNames;
 }
 
+const CloseIcon = () => (
+  <svg
```

Comment:
This doesn't need to be in the render of the function component.

Reply:
Out of curiosity: what's the difference in terms of where is it placed?

Reply:
It gets created every time if it in the function component. The way it is now, it is created once (the JS variable, not the Preact component), and then it is always referenced when being used as the `icon` prop.